<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adduser</class>
 <widget class="QDialog" name="adduser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>651</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../build-MY_store-Desktop-Debug/password_forgot.JPG</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter the username here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_nickname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter your childhood nick name here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_uncle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter name of you favorite uncle here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_friend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter the name of your best friend here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
